test/nim/src/classes/gdproptestnode.nim

```diff
@@ -34,76 +34,77 @@ MULTILINE-TEXT MULTILINE-TEXT MULTILINE-TEXT"""
   color_with_export_no_alpha*: Color = color(1, 1, 1)
 
 method onInit(self: PropTestNode) =
-  self.icon = ResourceLoader.load("res://icon.png") as gdref Texture2D
+#   self.icon = ResourceLoader.load("res://icon.png") as gdref Texture2D
```

Comment:
I commented this out because hot-reloading crashes after the second time due to this. Still need to investigate why. After the first hot-reload the icon texture is replaced by a magenta color and if you save the scene in that state it gets forever corrupted.